reana_commons/validation/utils.py

```diff
@@ -23,7 +26,29 @@
 from reana_commons.errors import REANAValidationError
 
 
-def validate_reana_yaml(reana_yaml: Dict) -> None:
+def _parse_schema_validation_error(error: ValidationError) -> List:
```

Comment:
Sounds right! While doing this, I was thinking that maybe it could be worth to put the validators that are considered not critical inside `config` (and maybe make them configurable by means of Helm chart or something like that) rather than as a local variable inside the function, but maybe it's too "low-level". What's your opinion on this?

Reply:
That's a good idea! But maybe something we can leave for later on, when/if we will decide to have more non-critical warnings?